config/kibana.yml

```diff
@@ -181,3 +181,5 @@
 # Maximum number of documents loaded by each shard to generate autocomplete suggestions.
 # This value must be a whole number greater than zero. Defaults to 100_000
 #unifiedSearch.autocomplete.valueSuggestions.terminateAfter: 100000
+
+#xpack.securitySolution.enableExperimental: ['expandableTimelineFlyoutEnabled']
```

Comment:
Is this change intentional? It is the first time I have seen a reference to `enableExperimental ` inside this file.

Reply:
no it's not, good catch! 😆